README.md

```diff
@@ -33,6 +33,15 @@ case:
 $ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
 ```
 
+## Minimum Supported Rust Version (MSRV)
+
+At the moment we use test and support the following Rust compiler versions:
+
+* On Ubuntu we test with the latest stable compiler version, as accessible through `rustup`.
+* On Fedora we test with the compiler version included with the Fedora 33 release.
+
+If you need support for other versions of the compiler, get in touch with us to see what we can do!
```

Comment:
Maybe mention somewhere that for those supproted versions we test on the CI that every PR works with them.

Reply:
I made a mistake on the line before the list above, will include it there.

Reply:
I was under the impression that we would use the packaged versions for the different distributions so in the case of ubuntu one would need to do the following: ```apt update && apt -y upgrade && apt install rustc && apt install cargo```. So that we could say that we support the latest version that the distribution provides using its update channels.

Reply:
In the Dockerfile we actually build with the pipe-to-shell command from the [Rust website](https://www.rust-lang.org/tools/install) - but that's a good point, maybe we should try out the one included in Ubuntu.

Reply:
It seems the latest supported is `1.43.0` (for `rustc`), but the weird thing is that `cargo` has version `0.44.1` 🤔

Reply:
> I was under the impression that we would use the packaged versions for the different distributions

I think we decided that as well for all the distributions that we want to support (like Fedora) but I think we should also definitely test with the latest stable Rust got from `rustup`. If Ubuntu is also one of the distribution we want to support then we should do the three:
* Ubuntu using the latest version of `cargo` from `apt`
* Fedora using the latest version of `cargo` from `dnf`
* *any distrib* using the latest version from `rustup`

My point being that for the case when we test on `rustup`, the distribution should not really matter.